react-props-and-expressions/my-app/src/App.jsx

```diff
@@ -0,0 +1,15 @@
+import './App.css';
+import CustomButton from './CustomButton';
+import './CustomButton.css';
```

Comment:
You would want to import this inside of your `CustomButton` component to keep all the logic confined inside there